### DISCUSSION TITLE
Clip Exporter

### DISCUSSION THREAD

**Discussion by TextAdventurer12** [ppy/osu#28347] [Ideas] [answered]
Sometimes I want to share snippets of gameplay, but right now the only way is to export the replay, convert it to an mp4, and then edit that mp4 in some other software. It would be really convenient to be able to select a portion of a replay (maybe a start/stop recording button in replay playback) and export it as an mp4.

**Comment by BlockHead007** [comment, marked as answer]
Not happening as mentioned in #27410
